### PR TITLE
Better support for subtechniques

### PR DESCRIPTION
**Pull request by Jack-McDowell**
Merits more testing, especially with monitor mode

### CODE REVIEW

**Comment by Jack-McDowell** [comment]
Squash this merge

File: BLUESPAWN-win-client/src/hunt/hunts/HuntT1037.cpp [hunk 1]
```diff
@@ -15,31 +15,40 @@ namespace Hunts {
         dwTacticsUsed = (DWORD) Tactic::Persistence | (DWORD) Tactic::PrivilegeEscalation;
     }
 
-    std::vector<std::shared_ptr<Detection>> HuntT1037::RunHunt(const Scope& scope) {
-        HUNT_INIT();
+    void HuntT1037::Subtechnique001(IN CONST Scope& scope, OUT std::vector<std::shared_ptr<Detection>>& detections){
+        SUBTECHNIQUE_INIT(001, Logon Script);
```
Comment (ION28) [inline BLUESPAWN-win-client/src/hunt/hunts/HuntT1037.cpp:19]: Change to [Logon Script (Windows]

**Comment by ION28** [comment]
Things I'd like fixed:

- [ ] Seeing several instances where detections have First Evidence at 1601-01-01 00:00:00.0Z

**Comment by Jack-McDowell** [comment]
> 
> 
> Things I'd like fixed:
> 
>     * [ ]  Seeing several instances where detections have First Evidence at 1601-01-01 00:00:00.0Z
> 
>     * [ ]  investigation to very long running T1055 (see AWS)

These are both issues that are unrelated to this PR. This PR is just to properly segment hunts. Those can be fixed separately.

**Pull request merged**